test/test_runner/csv_converter.cpp

```diff
@@ -118,27 +118,31 @@ void CSVToParquetConverter::readCopyCommandsFromCSVDataset() {
         auto path = std::filesystem::path(extractPath(tokens[3], '"'));
         auto table = tableNameMap[tokens[1]];
         table->csvFilePath = TestHelper::appendKuzuRootPath(path.string());
-        auto parquetFileName = path.stem().string() + ".parquet";
-        table->parquetFilePath = parquetDatasetPath + "/" + parquetFileName;
+        auto outputFileName = path.stem().string() + fileExtension;
+        table->outputFilePath = outputDatasetPath + "/" + outputFileName;
     }
 }
 
-void CSVToParquetConverter::createCopyFile() {
+void CSVConverter::createCopyFile() {
     readCopyCommandsFromCSVDataset();
-    auto parquetCopyFile =
-        LocalFileSystem::joinPath(parquetDatasetPath, std::string(TestHelper::COPY_FILE_NAME));
-    std::ofstream outfile(parquetCopyFile);
+    auto outputCopyFile =
+        LocalFileSystem::joinPath(outputDatasetPath, std::string(TestHelper::COPY_FILE_NAME));
+    std::ofstream outfile(outputCopyFile);
     if (!outfile.is_open()) {
         throw TestException(
-            stringFormat("Error opening file: {}, errno: {}.", parquetCopyFile, errno));
+            stringFormat("Error opening file: {}, errno: {}.", outputCopyFile, errno));
+    }
+    if (fileExtension == ".json") {
```

Comment:
Do we need to load json extension here? I can see we load twice, here and also in `convertCSVFiles`.

Reply:
Yes. One for csv->json conversions, one for json->internal importing once the tests run.